options/locale/locale_en-US.ini

```diff
@@ -175,6 +175,8 @@ app_url = Gitea Base URL
 app_url_helper = Base address for HTTP(S) clone URLs and email notifications.
 log_root_path = Log Path
 log_root_path_helper = Log files will be written to this directory.
+default_branch_name = Default branch name
+default_branch_name_helper = You can set default branch name (master, main, etc), leave empty if you do not care.
```

Comment:
```suggestion
default_branch_name_helper = Name of the default git branch.
```

Sounds more professional then that other text. Suggesting to leave empty was also not good.

Reply:
Absolutely same "leave empty" you can see upper on same page. Its professional, you can see examples of use. More professional do not change this. 

![image](https://user-images.githubusercontent.com/1711193/188030210-47ce70a5-c295-4c88-9ea8-01713c378adb.png)